python/cudf/benchmarks/API/bench_dataframe.py

```diff
@@ -121,10 +117,7 @@ def bench_groupby(benchmark, dataframe, num_key_cols):
     [
         "sum",
         ["sum", "mean"],
-        {
-            f"{string.ascii_lowercase[i]}": ["sum", "mean", "count"]
-            for i in range(6)
-        },
+        {f"{string.ascii_lowercase[i]}": ["sum", "mean", "count"] for i in range(6)},
```

Comment:
I find the previous version more readable, do we have to configure any line length settings to achieve that?

Reply:
I can re-configure ruff to go back to a `79` line length

Reply:
Of note: Some of our copyright lines lines are longer than 79 line length, so I had to `noqa` those.

Reply:
Thanks! I think `noqa`'ing those is fine.

Reply:
https://github.com/rapidsai/cudf/pull/15312#discussion_r1525563236

Adding noqa here isn't ideal. The SPDX identifiers are meant to be machine-readable...

Reply:
It doesn't appear that `ruff format` has any [options](https://docs.astral.sh/ruff/settings/#format) to specify text to ignore. I guess previously `black` ignored this since it wasn't code.

I think bumping the line length is the only way to avoid `noqa`s here

Reply:
This is a documented limitation:
https://github.com/astral-sh/ruff/issues/4429
https://github.com/astral-sh/ruff/issues/5899

☹️

Reply:
The reason I hesitate on this is that I think we are supposed to migrate towards SPDX identifiers in all our copyright headers. We shouldn’t noqa every file.

Reply:
FWIW I would also be OK with a line-length of 88 (that's what we use in pandas)

Reply:
We can, as the ruff FAQs suggest, run `ruff format` but ignore `E501` (line length issues). This _might_ lead to some lines that are over the line length, but with the conservative value of 79, I don't think it is necessarily that problematic.

Reply:
Good idea with ignoring `E501`. Yeah we can go with that for now.